docs/static/releasenotes.asciidoc

```diff
@@ -15,6 +15,25 @@ This section summarizes the changes in the following releases:
 [[logstash-8-1-0]]
 === Logstash 8.1.0 Release Notes
 
+[[known-issue-8-1-0]]
+==== Known issue ====
+
+* Uninstalling a plugin using `bin/logtash-plugin uninstall` may
+result in the following error:
+
+```
+Gem::LoadError: You have already activated jruby-openssl 0.12.2, but your Gemfile requires jruby-openssl 0.12.1. Prepending `bundle exec` to your command may solve this.
+```
+
+* Logstash should still run, and other plugin operation such as `update` and `install` should still work as expected.
+** `bin/logstash-plugin list` may fail with the same error after a failed uninstallation.
+* A successful plugin `update` will resolve this issue, and allow subsequent `uninstall` and `list` operations to
+work without issue.
+** Running `bin/logstash-plugin update logstash-filter-dissect` should mitigate this issue.
```

Comment:
Github gets confused because of the literal formatting inside the suggestion and blows up.  So here's the suggestion for cut-and-paste if you like it:  

Uninstalling a plugin using `bin/logtash-plugin uninstall` may
result in an error:

```
Gem::LoadError: You have already activated jruby-openssl 0.12.2, but your Gemfile requires jruby-openssl 0.12.1. Prepending `bundle exec` to your command may solve this.
```

Logstash should still run, and other plugin operations, such as `update` and `install`, should work as expected.

NOTE: The `bin/logstash-plugin list` command may fail with the same error after a failed uninstallation.

**Resolution**

A successful plugin `update` will resolve this issue, and allow subsequent `uninstall` and `list` operations to
work without issue.

To update a plugin, run `bin/logstash-plugin update logstash-<plugintype>-<plugin>` where `logstash-<plugintype>-<plugin>` is the name of the plugin. 
For example:`bin/logstash-plugin update logstash-filter-dissect`.

Reply:
Or, if you like the way it looks, and I can update the PR with a commit to adjust formatting. Lemme know.

Reply:
I like the format you suggested - if you have that ready to go, please update the PR with your suggestion at your convenience. Otherwise, I'll make the changes in the morning

Reply:
I updated formatting and changed the content a bit in the process. I interpreted `update logstash-filter-dissect` to be an example and formatted it as such.  If that's not the case, then we'll need to adjust the content.